packages/calcite-components/src/components/input-time-zone/input-time-zone.scss

```diff
@@ -0,0 +1,3 @@
+:host {
+  display: block;
+}
```

Comment:
I think this should import the `@include base-component();` mixin in order to support `:host([hidden]) {` I wonder why its working in the test now.

Reply:
Nice catch. 

I think that test helper needs some looking into.

Reply:
Never mind, the helper is working fine. Tests just need a-fixin'. 🔧